__main__.py

```
@@ -0,0 +1,600 @@
+            details = result.details
+            if 'flag' in str(details).lower() or 'ctf{' in str(details).lower():
+                print(f"  [🚩] Potential flag found!")
+            
+            # Show evidence score
+            if 'evidence_score' in details:
+                print(f"  [*] Evidence Score: {details['evidence_score']:.2f}")
+        else:
+            print(f"  [-] Failed: {plugin_name}")
+            # If test failed, skip exploitation
+            if 'Test' in plugin_name:
+                print(f"  [!] Skipping exploitation since test failed")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)